docker-compose.yaml

```diff
@@ -14,6 +14,9 @@ services:
       - 8081:8081
     volumes:
       - "./templates:/app/templates:ro"
+    depends_on:
+      - db
+    command: ["/wait-for-it.sh", "db:5432", "--", "/app/utask"]
```

Comment:
Could you just move the `command:` key just after the `build:` key (L.4) to be more consistent with the common way of writing a `docker-compose`?

Reply:
Done, thanks